setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
+universal=1
```

Comment:
It allows you to create universal wheels when you run:

`python setup.py sdist bdist_wheel register upload`